pygmt/clib/session.py

```diff
@@ -60,6 +61,7 @@
     "GMT_IS_PLP",  # items could be any one of POINT, LINE, or POLY
     "GMT_IS_SURFACE",  # items are 2-D grid
     "GMT_IS_VOLUME",  # items are 3-D grid
+    "GMT_IS_TEXT",  # Text strings which triggers ASCII text reading
```

Comment:
Is only ASCII supported for now? What about other encodings?

Reply:
GMT only accepts ASCII text strings. Any non-ASCII characters should be written in the form of octal codes.